js/tests/launchAsync.test.ts

```diff
@@ -144,30 +144,51 @@ describe('launchAsync tests', () => {
         const iframe = <HTMLIFrameElement>container.firstElementChild;
         expect(iframe.src.toLowerCase()).toMatch('https://foo.com/');
     });
+});
+
+describe('Utility method isValidSubdomain', () => {
+    it('should return false', () => {
+        expect(isValidSubdomain(null)).toBe(false);
+        expect(isValidSubdomain(undefined)).toBe(false);
+        expect(isValidSubdomain('')).toBe(false);
+        expect(isValidSubdomain('é')).toBe(false);
+        expect(isValidSubdomain('hasaccént')).toBe(false);
+        expect(isValidSubdomain('1é2')).toBe(false);
+        expect(isValidSubdomain('É')).toBe(false);
+        expect(isValidSubdomain('Ã')).toBe(false);
+        expect(isValidSubdomain('has space')).toBe(false);
+        expect(isValidSubdomain('has.period')).toBe(false);
+        expect(isValidSubdomain(' startswithspace')).toBe(false);
+        expect(isValidSubdomain('endswithspace ')).toBe(false);
+        expect(isValidSubdomain('-startswithdash')).toBe(false);
+        expect(isValidSubdomain('endswithdash-')).toBe(false);
+    });
+
+    it('should return true', () => {
+        expect(isValidSubdomain('valid')).toBe(true);
+        expect(isValidSubdomain('valid10with2numbers')).toBe(true);
+        expect(isValidSubdomain('1234')).toBe(true);
+    });
+});
 
-    describe('Utility method isValidSubdomain', () => {
-        it('should return false', () => {
-            expect(isValidSubdomain(null)).toBe(false);
-            expect(isValidSubdomain(undefined)).toBe(false);
-            expect(isValidSubdomain('')).toBe(false);
-            expect(isValidSubdomain('é')).toBe(false);
-            expect(isValidSubdomain('hasaccént')).toBe(false);
-            expect(isValidSubdomain('1é2')).toBe(false);
-            expect(isValidSubdomain('É')).toBe(false);
-            expect(isValidSubdomain('Ã')).toBe(false);
-            expect(isValidSubdomain('has space')).toBe(false);
-            expect(isValidSubdomain('has.period')).toBe(false);
-            expect(isValidSubdomain(' startswithspace')).toBe(false);
-            expect(isValidSubdomain('endswithspace ')).toBe(false);
-            expect(isValidSubdomain('-startswithdash')).toBe(false);
-            expect(isValidSubdomain('endswithdash-')).toBe(false);
-        });
-
-        it('should return true', () => {
-            expect(isValidSubdomain('valid')).toBe(true);
-            expect(isValidSubdomain('valid10with2numbers')).toBe(true);
-            expect(isValidSubdomain('1234')).toBe(true);
-        });
+const fs = require('fs');
+describe('Verify SDK version is valid', () => {
+
+    it('check version', () => {
+        const packageJson: string = fs.readFileSync("package.json", "utf8");
+        const sdkVersion: string = JSON.parse(packageJson).version;
+        console.log(`SDK version: ${sdkVersion}`);
```

Comment:
Is there a reason you choose to log to console here? If not, it may be better to remove this logging line

Reply:
For debugging. Can be removed.

Reply:
Though, if a test ever fails due to an invalid version, we'll have the version traced straight away in the test log, which simplifies debugging.

Reply:
Can remove if you feel strongly

Reply:
Ok sure - it threw me off because we don't log in another test in this file but I see your point, I'm fine with keeping it